src/util.cpp

```diff
@@ -456,6 +464,35 @@ void ArgsManager::ParseParameters(int argc, const char* const argv[])
             m_override_args.erase(it);
         }
     }
+    return true;
+}
+
+bool ArgsManager::IsArgKnown(const std::string& key, std::string& error)
+{
+    bool found = false;
+    for (const auto& arg_map : m_available_args) {
+        for (const auto& arg : arg_map.second) {
+            std::string arg_name = arg.first;
+            size_t eq_index = arg_name.find('=');
```

Comment:
If you'd move this "="-splitting to `AddArg` instead, and would store the second part in `Arg` rather than in the key, this entire loop could be avoided and replaced with a map lookup.

Reply:
Done.